src/Events/DietcubeEventAbstract.php

```diff
@@ -1,14 +1,11 @@
 <?php
-/**
- *
- */
 
 namespace Dietcube\Events;
 
 use Dietcube\Application;
 use Symfony\Component\EventDispatcher\Event;
 
-class DietcubeEventAbstract extends Event
+abstract class DietcubeEventAbstract extends Event
```

Comment:
It's expected an abstract class. 
It's one of the big breaking changes from the current DietCube architecture.